public/js/app.js

```diff
@@ -1,4 +1,4 @@
-angular.module('mean', ['ngCookies', 'ngResource', 'ui.bootstrap', 'ui.route', 'mean.system', 'mean.directives'])
+angular.module('mean', ['ngCookies', 'ngResource', 'ui.bootstrap', 'ui.route', 'ngRoute', 'mean.system', 'mean.directives'])
```

Comment:
@CodeMuhammed please refactor such that each of this includes is on it's own line and also resolve the merge conflicts so we can merge properly, thanks.